GTScript-Parallel-model.md

```diff
@@ -103,3 +103,132 @@ for k in range(start, end):
         u[i, j, k] = 1
     parfor [i_start:i_end, j_start:j_end]:
         b[i, j, k] = u[i-2,j,k] + u[i+1,j,k] + u[i,j-1,k] + u[i,j-2,k]
+```
+
+## Conditionals
+
+GTScript supports 3 kinds of conditionals:
+
+- compile-time conditionals using `with CONDITION()` (former `if __INLINED`)
```

Comment:
We were hoping to remove `__INLINED`, and this seems compatible with that, while making it clear that it's a compile-time conditional. Nice!